test/plugin_functional/test_suites/core_plugins/rendering.ts

```diff
@@ -402,7 +403,7 @@ export default function ({ getService }: PluginFunctionalProviderContext) {
         'telemetry.labels.branch (string)',
         'telemetry.labels.ciBuildId (string)',
         'telemetry.labels.ciBuildJobId (string)',
-        'telemetry.labels.ciBuildNumber (number)',
```

Comment:
Are we intentionally removing `ciBuildNumber` here?

Reply:
oops, great catch! It's a copy-paste bloops

Reply:
it's [back](https://github.com/elastic/kibana/pull/179675/files#diff-eb6bd8a02c024aa1ab5c3debbeda149e8d2fb2614bca80d09c3f9aa21100ed7dR179) 🪄